### PR TITLE
[Component with its own class name as service type ] LPS-196226 Refactor Registry classes from multiple modules

### PR DESCRIPTION
**Pull request by MrJohn1911**
Ticket: https://liferay.atlassian.net/browse/LPS-196226

### CODE REVIEW

**Comment by MrJohn1911** [comment]
ci:test:startup

**Comment by Kvale1733** [comment]
ci:test:commerce

**Comment by MrJohn1911** [comment]
ci:test:relevant

**Comment by Kvale1733** [comment]
@MrJohn1911 No regressions found.

File: modules/apps/frontend-js/frontend-js-bundle-config-extender/src/main/java/com/liferay/frontend/js/bundle/config/extender/internal/JSBundleConfigRegistryUtil.java [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+			}
+		}
+
+	}
+
+	static {
+		Bundle bundle = FrameworkUtil.getBundle(
+			JSBundleConfigRegistryUtil.class);
+
+		_bundleContext = bundle.getBundleContext();
```
Comment (rafaprax) [inline modules/apps/frontend-js/frontend-js-bundle-config-extender/src/main/java/com/liferay/frontend/js/bundle/config/extender/internal/JSBundleConfigRegistryUtil.java:131]: Hey @MrJohn1911, please remove the `_bundleContext` static field in favor of using an instance field in the  JSBundleConfigServiceTrackerCustomizer.

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by MrJohn1911** [comment]
ci:test:relevant

**Comment by Kvale1733** [comment]
ci:test:stable

**Comment by catarineaguiar** [comment]
no regressions found. SF issue is not related

**Comment by shuyangzhou** [comment]
@MrJohn1911 @rafaprax for the 1st commit, please change it into a ServiceTrackerMap usage first. Then continue on the change to make it an util.
Please resend.

**Pull request closed without merging**